### DISCUSSION TITLE
Mac OS installation steps

### DISCUSSION THREAD

**Discussion by ghost** [AlexandreRouma/SDRPlusPlus#472] [Q&A]
Hi, I am not a Linux geek. I follow this app development since a while and I am looking forward to install it on my MacBook M1 processor. 
Any ideas when a complete install packaged will be available for Mac OS?

Great works guys, very promising. 

73 de Jeff | VA2SS

**Comment by AlexandreRouma** [comment]
Hello,

I unfortunately do not own or have access to a Mac so I won't be able to add that myself.
There will only be an install package when someone else contributes one

Regards

**Comment by ghost** [comment]
Ok I truly  understand. I suggest you change your statement for Mac OS in your introduction « please wait a few more days for me to fix that » with something else more relevant  asking at least for people contribution to fix that.

**Comment by n2iw** [comment]
I was able to compile SDR++ on my M1 Mac min about a month ago. The key is to use MacPorts instead of HomeBrew to install dependencies, because there are still some key packages don't support M1 in HomeBrew.

Here are commands I used to compile it:

Download and install MacPorts
Download and install SDRPlay API for M1

sudo port install cmake volk glew glfw airspy airspyhf hackrf rtl-sdr soapysdr libiio libad9361-iio rtaudio portaudio pandoc limesuite

git clone https://github.com/AlexandreRouma/SDRPlusPlus.git
cd SDRPlusPlus
mkdir -p build
cd build
cmake .. -DOPT_BUILD_SDRPLAY_SOURCE=ON -DOPT_BUILD_LIMESDR_SOURCE=ON
make
sudo make install
sdrpp

**Reply by ruhid** [reply]
Hi, cmake cant find volk location.  How can İ point it ?




- Checking for module 'volk'
--   No package 'volk' found
CMake Error at /opt/local/share/cmake-3.19/Modules/FindPkgConfig.cmake:556 (message):
  A required package was not found
Call Stack (most recent call first):
  /opt/local/share/cmake-3.19/Modules/FindPkgConfig.cmake:744 (_pkg_check_modules_internal)
  core/CMakeLists.txt:73 (pkg_check_modules)

**Reply by n2iw** [reply]
Try run "sudo port install volk", it was port of my commands. It probably failed for some reason, run that command and see what error it gives you.

**Reply by ghost** [reply]
Thanks to all who replied…. But a clearer step by step process would be appreciated….. for all people like me who are not Linux or 3rd parties side load libs command geek on Mac OS.

I really wish to try it…. I am waiting to order a RF output board for my IC-7300…. Because there is not much software solution running on Mac platform…. All other software running on Mac are disappointing … never get updated since a while most of them. 

Thanks all for your contribution.

**Comment by loewal** [comment]
Did you try:

• Sdrangel
• Cubicsdr
• Gqrx

?

Loe Walter
Insula 103
6416 BZ  Heerlen
☎️ +31-6-24476963
